contracts/token/ERC721/ERC721.sol

```diff
@@ -269,34 +280,37 @@ abstract contract ERC721 is Context, ERC165, IERC721, IERC721Metadata, IERC721Er
      *
      * Emits a {Transfer} event.
      */
-    function _mint(address to, uint256 tokenId) internal virtual {
+    function _mint(address to, uint256 tokenId) internal {
         if (to == address(0)) {
             revert ERC721InvalidReceiver(address(0));
         }
-        if (_exists(tokenId)) {
```

Comment:
Yeah I don't really know why this function exists (:speak_no_evil:). Overriding it should have no effect either.

I'd be in favor of keeping `_requireMinted` only.

Reply:
Yeah I don't really know why this function exists (:speak_no_evil:). Overriding it should have no effect either.

I'd be in favor of keeping `_requireMinted` only.

Reply:
Yeah let's do `_requireMinted` only, it's the same.

I would suggest `_checkMinted` to be consistent with the `_check` convention. Not strong about it but perhaps you might like it.

Reply:
I guess there was a point to if when you wanted to do different logic (like calling `_mint` instead of `_transfer`) based on some logic. Also, at some point we did not have `_ownerOf` but only the public `ownerOf` that revert is 0 ... so that was literraly the only way to know if a token is minted or not that would not revert if it does not exist.

Fells like its no longer needed, so I'm removing it.